src/components/Settings/Settings.tsx

```diff
@@ -0,0 +1,73 @@
+import { Box } from "@mui/material";
+import TabPanel from "components/ui/TabPanel/TabPanel";
+import Tabs from "components/ui/Tabs/Tabs";
+import { TabType } from "models/ui.model";
+import { FC, useState, SyntheticEvent } from "react";
+import { useNavigate, useLocation } from "react-router-dom";
+import EditProfile from "./EditProfile";
+
+const TABS: TabType[] = [
+  {
+    label: "Edit Profile",
+  },
+  {
+    label: "Professional Account",
+  },
+  {
+    label: "Change Password",
+  },
+];
+
+const tabNameToIndex: { [key: number]: string } = {
+  0: "/accounts/edit",
+  1: "/accounts/professional_account_settings",
+  2: "/accounts/change_password",
+};
+
+const indexToTab: { [key: string]: number } = {
```

Comment:
You don't need two objects. Try to get key by value like this

`function getKeyByValue(object, value) {
  return Object.keys(object).find(key => object[key] === value);
}`